.circleci/config.yml

```diff
@@ -26,9 +28,29 @@ common: &common
     - run:
         name: install dependencies
         command: pip install --user tox
+    - run:
+        name: check for solc
+        command: |
+          if [[ -n "$SOLC_VERSION" ]]; then 
+            echo "$SOLC_VERSION"
+            sudo python3 -m pip install -U "py-solc>=1.2.0"
+            sudo python3 -m solc.install $SOLC_VERSION
+            echo 'export SOLC_BINARY=/root/.py-solc/solc-$SOLC_VERSION/bin/solc' >> $BASH_ENV
+            echo 'export LD_LIBRARY_PATH=/root/.py-solc/solc-$SOLC_VERSION/bin' >> $BASH_ENV
+            source $BASH_ENV
+            sudo chmod a+rwx $SOLC_BINARY
+            sudo chmod a+rwx $LD_LIBRARY_PATH
+            sudo $SOLC_BINARY --version
+          fi
     - run:
         name: run tox
-        command: ~/.local/bin/tox
+        command: |
+          if [[ -n "$SOLC_VERSION" ]]; then
+            echo "$SOLC_BINARY"
+            sudo $SOLC_BINARY --version
+            $SOLC_BINARY --version
```

Comment:
Shouldn't the permissions set on 41 - `all can read, write, execute`, allow 51 to run? 50 is running fine. I've also tried `chmod -R 755 ...` on 41 which didn't help